src/app/garfile/home/index.njk

```diff
@@ -34,7 +34,7 @@
                   Please select how you would like to create your GAR to proceed.
                 </span>
                 {{ m.error_message("garoption", errors) }}
-                  <div class="govuk-radios govuk-radios--inline" id="garoption">
+                  <div class="govuk-radios" id="garoption">
```

Comment:
Nigel, When I look at the page, the text I see, associated with radio buttons is not what I find in prototype page https://jhvxrb3.egar-staging.homeoffice.gov.uk/version3/create-gar-landing ..ie.. "Upload a file", "Online" instead of "create a.." as in done here. Is there a reason for that?

Reply:
Hi

As I mentioned in the notes in the comments above the actual text for the radio buttons seems to be passed into the page via the back end in the garoptions object which I don't think I have access to @pawniemiec can you confirm this is the case.

Reply:
Hi

I have found where the data has come from and I can update by the looks so I'll do that.

I had assumed it was coming from the database or api sorry!

Reply:
Seed data for radio options pushed.

Reply:
Its "Upload a file" as there in https://jhvxrb3.egar-staging.homeoffice.gov.uk/version3/create-gar-landing.

Reply:
Also the story says, it needs to have an end to end test created as well. Once you finish your part, may be I will create a separate story for e2e test.

Reply:
Sorry updated